### DISCUSSION TITLE
How to set (change) "Default Value" of plugin parameter

### DISCUSSION THREAD

**Discussion by kovacm** [tinymce/tinymce#8309] [General]
I am using "Image" plugin. I would like to change "Default Value" for option "image_caption":

https://www.tiny.cloud/docs-4x/plugins/image/#image_caption

By default, "Default Value" is "false". I would like to change "Default Value" to be "true" (I would like to each image have caption turned on).

How to do that?

I expect to each image have caption turned on by default.

**Comment by lostkeys** [comment]
Hi @kovacm. This is not supported unfortunately. To help us gain insight and prioritize the feature request I have this standard deck of questions I'd love for you to answer. If you don't want to do it publicly, feel free to submit your feedback over at our [roadmap website](https://www.tiny.cloud/roadmap/)

1. Why is the requested feature important for your app/service?
2. What is the consequence of not having the requested feature?
3. How important would you rate the requested feature choosing from nice-to-have, important or critical?

Thanks,
Fredrik – Product Manager

**Comment by kovacm** [comment]
Thank you for replay.

1. In my case, I would like to every inserted image already have a caption added. But in broader sense - it would be nice to have ability to set default options in plugins.
2. No major consequence in my case.
3. I would say it is important to have this ability. Probably there are many cases where ability to set default value for plugin options would be crucial.

**Reply by lostkeys** [reply]
Thank you for the additional details, I've added it to the records for product future considerations.
